packages/suite-data/files/translations/cs.json

```diff
@@ -254,6 +254,7 @@
   "TR_ADDRESS_FORMAT": "Správný formát adresy",
   "TR_ADDRESS_MODAL_CLIPBOARD": "Kopírovat adresu",
   "TR_ADDRESS_MODAL_TITLE": "{networkName} adresa pro příjem",
+  "TR_ADDRESS_PHISHING_WARNING": "Abyste zabránili phishingovým útokům, měli byste ověřit adresu v zařízení Trezor. {claim}",
```

Comment:
that will not work, next translation sync would override this. translations should not be changed manually in .json files.

Reply:
I see, ok thanks, and how do you handle any change in translations then please?

Reply:
We manage the translations in Crowding https://crowdin.com/project/trezor-suite, I will put there this message, thanks!

Reply:
So should I revert this change then?